app/controllers/idv/review_controller.rb

```diff
@@ -63,7 +75,7 @@ def idv_params
     end
 
     def phone_confirmation_required?
-      !idv_params[:phone_confirmed_at] || idv_params[:phone] != current_user.phone
+      idv_params[:phone] != current_user.phone
```

Comment:
Good question.

As I read `Idv::PhoneForm.update_idv_params` it seems like this check is, indeed, superfluous. I think what I was originally imagining was the edge case where the 2fa phone number was set but not confirmed (somehow) on the `User`. I don't think it is possible to reach the IdV process, though, w/o first having confirmed 2fa, so your change seems ok as-is.